### PR TITLE
mergify: two-day delay should account for the squash label

### PR DESCRIPTION
**Pull request by ulysses4ever**
That was an oversight on my part when I did it initially: the squash label was overseen.

### CODE REVIEW

**Review by Mikolaj — APPROVED** [review]
Never-ending story. :) I've heard they've even made a film about that. How did the video about cabal's usage of mergify go?

**Comment by ulysses4ever** [comment]
@ffaf1 any chance I could get a review from you? 😊

**Comment by ulysses4ever** [comment]
@mergify rebase

**Review by ffaf1 — APPROVED** [review]
let’s see where this brings us!

**Comment by ulysses4ever** [comment]
Has to be merged manually because it's a change in the config.

**Comment by ulysses4ever** [comment]
Oh wow, mergify says it's an invalid config, but I literally took it from the docs. I guess, I'd need to ask in their tracker.

**Comment by ulysses4ever** [comment]
As usual, it was me who was wrong, not Mergify. Thanks upstream for pointing out to the error.

Will merge once all checks are passed.

**Pull request merged**
